charts/kcp/templates/etcd-statefulset.yaml

```diff
@@ -10,9 +10,9 @@ spec:
   clusterIP: None
   ports:
     - port: 2379
-      name: client
+      name: https-client
     - port: 2380
-      name: peer
+      name: https-peer
```

Comment:
Can we leave a comment that the "https-" prefix is now a critical part of this template and must never be changed or removed ever, because of Istio?

Reply:
actually that's a very solid point. I forgot about `appProtocol`. Let me test that.